src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -142,7 +143,9 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, reimbursementAcc
     const hasMembersError = PolicyUtils.hasEmployeeListError(policy);
     const hasPolicyCategoryError = PolicyUtils.hasPolicyCategoriesError(policyCategories);
     const hasGeneralSettingsError = !isEmptyObject(policy?.errorFields?.generalSettings ?? {}) || !isEmptyObject(policy?.errorFields?.avatarURL ?? {});
-    const shouldShowProtectedItems = PolicyUtils.isPolicyAdmin(policy);
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
+    const employee = policyProp?.employeeList?.[currentUserPersonalDetails.login ?? ''] ?? {};
+    const shouldShowProtectedItems = PolicyUtils.isPolicyAdmin(policyProp, employee.role);
```

Comment:
Why do you pass the role to the method? I think you should pass current user account id.